rms/auth/services.py

```diff
@@ -0,0 +1,37 @@
+import uuid
+from datetime import datetime, timedelta
+import bcrypt
+
+from fastapi import HTTPException, Response, status
+from pydantic import BaseModel
+
+from rms.auth.managers import UserManager, UserCookieManager
+from rms.auth.models import UserCookie
+
+
+class LoginRequest(BaseModel):
+    email: str
+    password: str
+
+
+def validate_credentials(email: str, password: str):
```

Comment:
pls make this functions return type something like `tuple[User, UserCookie] | None`